alibi_detect/cd/tabular.py

```diff
@@ -106,6 +107,11 @@ def __init__(
                                  'Dict[int, NoneType], Dict[int, int], Dict[int, List[int]]')
             self.x_ref_categories = categories_per_feature
             self.cat_vars = list(self.x_ref_categories.keys())
+        # No categories_per_feature dict so assume no categorical features present
+        else:
+            self.x_ref_categories, self.cat_vars = {}, []
+            warnings.warn('No `categories_per_feature` dict provided so all features are assumed to be categorical. '
```

Comment:
"... all features are assumed to be numerical." instead of categorical.

Reply:
Ooops, good spot, thanks!

Reply:
Corrected now! 👍🏻